templates/taxi/car_list.html

```diff
@@ -1,7 +1,14 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+    <h1>
+    Car list
+    <a
+        style="float: center"
+        href="{% url 'taxi:car-create' %}"
+    > <input class="btn btn-primary" type="submit" value="+">
```

Comment:
Do not break the tag in this way, place arguments on the same line with that tag.